### PR TITLE
Add 3raxton to _data

### PR DESCRIPTION
**Pull request by 3raxton**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/3raxton/3raxton.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Braxton Huff
+institution: Santa Monica College
+profile_pic: braxton.png 
+quote: Give back and support those who helped you make it to where you are today – braxtonhuff.com
```
Comment (Mohammed-Shoaib) [inline _data/3raxton/3raxton.md:5]: Change the em dash – to a single dash - as it is a non-ascii character.

Comment (3raxton) [reply _data/3raxton/3raxton.md]: Thanks for making me aware of this. I've updated it in [this commit](https://github.com/education/graduation/pull/4298/commits/6f9f74cfd0dc339a568d49bf7cea28290b93c782). Please let me know if there is anything else needed!

**Comment by 3raxton** [comment]
Thanks for helping me out @Mohammed-Shoaib, 

There is one question I have for you. When reading the README it seemed like the picture needed to be exactly 544x544 px. Can it be larger?

Aside from the rest of the checklist is complete.

**Comment by Mohammed-Shoaib** [comment]
@3raxton Happy to help :blush:

Good question! The size of the image must be **at least** 544x544 (pixels), so yes it can be larger.

**Comment by 3raxton** [comment]
Thanks for letting me know @Mohammed-Shoaib! I've just updated my picture [here](https://github.com/education/graduation/pull/4298/commits/7e09a4e37f17feb3778b733f8e3612d787cd20c6) to make it a better resolution. Would you be opposed to the README.md for clarity on the size of the image for future contributors?

**Comment by Mohammed-Shoaib** [comment]
@3raxton The new image looks perfect :100:
I do believe this was explained in an issue which can be found [here](https://github.com/education/graduation/issues/805#issuecomment-632758736)
> The profile picture NEEDS to be squared (1:1 or width=height) and has at least 544 pixels :framed_picture: 

But yes, adding it to the README file might be better for future contributors. I don't want to steal your thunder, would you prefer making a pull request by adding the keyword _atleast_ in the README file?

**Comment by 3raxton** [comment]
@Mohammed-Shoaib I'm appreciative of the way you're approaching this. Should I make the change in this pull request or would it be better to do it somewhere else?

**Comment by Mohammed-Shoaib** [comment]
@3raxton Create a separate pull request! Also, to not confuse the maintainers with other PRs it would be nice if you added some labels or tags like "suggestion", "improvement", or anything you feel describes it well.

Thank you for contributing! :heart:

**Comment by 3raxton** [comment]
Sounds like a plan @Mohammed-Shoaib! I'm still seeing that merging is blocked for this PR, is there anything further I need to do?

**Comment by Mohammed-Shoaib** [comment]
Hey @3raxton,

Your pull request is perfect! :confetti_ball: I approved the changes. :+1:

However, I cannot merge it as I do not have write access. You'll have to wait for a reviewer with _write_ access to merge your pull request.

Till then sit back, relax, and enjoy :relaxed: 

Best,
Mohammed Shoaib

**Pull request merged**